CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to Merlin will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Changed
+- Task id is now the path to the directory
```

Comment:
Let's just move this to the 1.12.2b1 release notes as I'm going to wait to release it until this PR and #484 are merged.